lib/eventFactory.js

```diff
@@ -206,7 +207,9 @@ function createBuilds(config) {
         .then((jobsToStart) => {
             // No jobs to start (eg: when jobs are disabled or startFrom is not valid)
             if (jobsToStart.length === 0) {
-                throw new Error('No jobs to start');
+                winston.info('No jobs to start.');
```

Comment:
Can we log out the `eventId` together with this line? that will be easier to track later on

Reply:
@minz1027 
That makes sense, I added `eventId` in the log. thanks!